config/build-test-translations.js

```
@@ -0,0 +1,52 @@
+/* eslint-disable no-console */
+const fs = require("fs");
+const path = require("path");
+const childProcess = require("child_process");
+
+const rootDir = process.cwd();
+const i18nDir = path.join(rootDir, "i18n");
+const outputFile = path.join(rootDir, "client", "test-translation.json");
+
+//eslint-disable-next-line consistent-return
```

ESLint

[error] Expected exception block, space or tab after '//' in comment.

(spaced-comment)